examples/ui/text_debug.rs

```diff
@@ -19,8 +20,8 @@ fn main() {
 
 struct TextChanges;
 
-fn infotext_system(mut commands: Commands, asset_server: Res<AssetServer>) {
-    let font = asset_server.load("fonts/FiraSans-Bold.ttf");
+fn infotext_system(mut commands: Commands) {
+    let font = DEFAULT_FONT_HANDLE.typed();
```

Comment:
I think this variable is no longer needed.

Reply:
if you want to change something back to that font youd need it

Reply:
Can replace its `TextStyle` with a style using `..TextStyle::Default`?

Reply:
i guess? i don't really see why it matters / quad_handle works the same way, why not just leave it exposed

Reply:
I guess it's just that the mechanism is *weird*, especially the `.typed()` thing

I'm not saying not to leave it exposed, I'm more suggesting not to use it in the example.

Reply:
ah, fair point